site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/repository/EntityStore.java

```diff
@@ -0,0 +1,14 @@
+package uk.ac.ox.ndph.mts.site_service.repository;
+
+/**
+ * Interface for a store of a data model entity
+ */
+public interface EntityStore<T> {
```

Comment:
not sure why this is named as a Store? it acts a service to the FhirRepo? Couldnt it be called FhirService or similar?

Reply:
Taking forward as tech debt.